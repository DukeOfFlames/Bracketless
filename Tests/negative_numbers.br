START
째 i_11 = 5
째 i_12 = -5
째 i_21 = 5/1
째 i_22 = -5/1
#drucke{i_11 + i_12}
#drucke{i_21 + i_12}
#drucke{i_11 + i_22}
#drucke{i_21 + i_22}
END