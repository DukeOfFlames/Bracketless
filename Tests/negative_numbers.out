[Integer:
  0]
[Float:
  0.0]
[Float:
  0.0]
[Float:
  0.0]
